README.md

```diff
@@ -195,6 +195,10 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [WPHUB](https://www.wphub.com/write-for-us/) - $200 per piece
   > Wordpress tutorials and articles.
+
+- [Enlear](https://www.enlear.com/) - Vary per writer experiance
```

Comment:
there's  a typo on experience.. also could you post the ranges for beginner and advanced writers?